tests/wamr-test-suites/spec-test-script/multi_module_ignore_cases.patch

```diff
@@ -94,7 +100,19 @@ index 994e0f4..8fbcc02 100644
  
  (assert_unlinkable
    (module (import "Mg" "mut_glob" (global i32)))
-@@ -130,7 +130,7 @@
+@@ -93,6 +96,7 @@
+ )
+ 
+ 
++(; aot doesn't support to import globals
```

Comment:
Had better make another patch for aot multi module, e.g. create a small patch to disable this case based on the patch for interpreter multi module, so here the case can be tested in interpreter mode.